libs/core/src/components/sage-switch/test/sage-switch.spec.tsx

```diff
@@ -0,0 +1,182 @@
+import { newSpecPage } from '@stencil/core/testing';
+import { SageSwitch } from '../sage-switch';
+
+describe('sage-switch', () => {
+  it('renders an input as a checkbox with label', async () => {
+    const page = await newSpecPage({
+      components: [SageSwitch],
+      html: `
+        <sage-switch
+          component-id="sage-switch-e1"
+          label="Switch label">
+        </sage-switch>
+      `
+    });
+    expect(page.root).toEqualHtml(`
```

Comment:
missing newline at 15...I struggle with this. But maybe here is where we can make a choice. I noticed down below you have a linebreak after the `const`, but here you don't. Personally, I prefer the extra line of separation, but that is my opinion. 

In this case, I'll just ask for consistency within the file.